terraform-modules/k8s/k8s-cluster.tf

```diff
@@ -65,7 +65,7 @@ resource "google_container_cluster" "cluster" {
   private_cluster_config {
     enable_private_endpoint = var.enable_private_endpoint
     enable_private_nodes = var.enable_private_nodes
-    master_ipv4_cidr_block = var.private_master_ipv4_cidr_block
+    master_ipv4_cidr_block = var.master_ipv4_cidr_block
```

Comment:
Are you sure this is the right fix? The k8s config Monster has been using has different values for two variables.

Is the real bug that `master_ipv4_cidr_block` should be used at the top-level somewhere, but it isn't?

Reply:
0.12 refactor var.master_ipv4_cidr_block was defined and not used. Before the 0.12 it was used as in this PR. 

https://github.com/broadinstitute/terraform-shared/blob/ms-k8fix2/terraform-modules/k8s/variables.tf#L31-L33

Reply:
Based on the diff, var.master_ipv4_cidr_block wasn't used before the refactor: https://github.com/broadinstitute/terraform-shared/commit/f2556b66cab23ec781b92a82535e3652964e1c27

var.private_master_ipv4_cidr_block didn't exist. I created it to replace the old var.private_cluster_config which was schema-less.

I don't mind dropping one of the vars, but I think it's better to drop the old var.master_ipv4_cidr_block and force people to think about setting the new var, instead of changing the behavior of the existing var from being unused to having an effect.